RS Dashboard: Get started option (#568)
rudderlabs/rudderstack-docs: get-started/index.mdx

Follow these steps to set up a connection in RudderStack:

1. Sign up for [RudderStack Cloud](https://app.rudderstack.com/signup?type=freetrial).

> 2. You can then view the **Get Started** screen on RudderStack dashboard.

Reviewer: ```suggestion
2. You will be presented with the **Get Started** page on RudderStack dashboard, as shown:
```
